app/assets/stylesheets/views/_guide.scss

```diff
@@ -11,4 +11,17 @@
     margin-left: 1.5em;
     padding-left: 0.3em;
   }
+
+  // FIXME: Remove when typography improvements made to
+  //        govspeak component
+  //
+  // Distinguish between part titles (27px) and h2s
+  // within content of part by reducing font-size
+  //
+  // Override h2 sizes to match frontend typography
```

Comment:
can we say 'alphagov/frontend' or something specific for people that are not in the know?